common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
   "cloth_config.voicechat.config.audio_packet_threshold": "Audio packet threshold",
-  "cloth_config.voicechat.config.audio_packet_threshold.description": "The maximum amount of audio packets that are held back if a packet arrives out of order or gets dropped\nThis prevents discarding audio packets that are slightly out of order\nSet this to 0 to disable",
-  "cloth_config.voicechat.config.voice_deactivation_delay": "Deactivation delay",
-  "cloth_config.voicechat.config.voice_deactivation_delay.description": "The time it takes for the microphone to deactivate when using voice activation\nA value of 1 means 20 milliseconds, 2=40 ms, 3=60 ms, ...",
+  "cloth_config.voicechat.config.audio_packet_threshold.description": "The maximum number of audio packets that should be held back if a packet arrives out of order or is dropped.\nThis prevents audio packets that are only slightly out of order from being discarded.\nSet this to 0 to disable.",
+  "cloth_config.voicechat.config.voice_deactivation_delay": "Microphone deactivation delay",
+  "cloth_config.voicechat.config.voice_deactivation_delay.description": "The time it takes for the microphone to deactivate when using voice activation.\nA value of 1 means 20 milliseconds, 2=40 ms, 3=60 ms, and so on.",
   "cloth_config.voicechat.config.output_buffer_size": "Output buffer size",
-  "cloth_config.voicechat.config.output_buffer_size.description": "The size of the audio output buffer in packets\nHigher values mean a higher latency, but less crackles\nIncrease this value if you have an unstable internet connection",
-  "cloth_config.voicechat.config.hud_icon_scale": "HUD icon scale",
-  "cloth_config.voicechat.config.hud_icon_scale.description": "The scale of the HUD icons",
-  "cloth_config.voicechat.config.hud_icon_pos_x": "HUD icon X pos",
-  "cloth_config.voicechat.config.hud_icon_pos_x.description": "The X position of the HUD icons\nNegative values mean anchoring to the right",
-  "cloth_config.voicechat.config.hud_icon_pos_y": "HUD icon Y pos",
-  "cloth_config.voicechat.config.hud_icon_pos_y.description": "The Y position of the HUD icons\nNegative values mean anchoring to the bottom",
-  "cloth_config.voicechat.config.group_player_icon_orientation": "Group HUD icon orientation",
-  "cloth_config.voicechat.config.group_player_icon_orientation.description": "The orientation of the player icons in the group HUD",
+  "cloth_config.voicechat.config.output_buffer_size.description": "The size of the audio output buffer (in packets).\nHigher values mean a higher latency but less crackling.\nIncrease this value if you have an unstable Internet connection.",
+  "cloth_config.voicechat.config.hud_icon_scale": "Voice chat HUD icon scale",
+  "cloth_config.voicechat.config.hud_icon_scale.description": "The scale of the icons in the voice chat HUD, such as microphone status, connection status, or speaking style indicator.",
```

Comment:
Whats a speaking style indicator?

Reply:
Your reaction clearly shows that the phrase I came up with to describe this thing isn't clear enough. If it is causing trouble for you, the person who knows the most about this mod, what can we expect from regular users?

But what exactly is this "thing"? Since I couldn't think of anything better, I used the phrase "speaking style indicator" to refer to the two icons in the HUD depicting a small and a large microphone that change their form depending on your intent.

Do you have any suggestions on what I should do here? Please let me know.

Reply:
Wouldn't "microphone status" already cover this case?

Reply:
You're probably right. I overcomplicated everything by coming up with a third element for the list just to make the list longer. Sometimes, the simplest solution is the best one.

I'm sorry for wasting your time here.